docs/src/intervention.html.jinja2

```diff
@@ -557,6 +557,34 @@
                         </div>
                     </div>
 
+                    <div class="row">
+                        <div class="col-md-3">
+                            <label>Discharge</label>
+                            <div class="form-group">
+                                <label for="rbrasdt">Date</label>
+                                {{ f.datepicker("rbrdd", "past") }}
+                            </div>
+                        </div>
+                        <div class="col-md-3">
+                            <label>Readmit within 30 days of discharge?</label>
+                            <div class="form-group">
+                                {{ f.inlineRadio("rbrthd", "rbrthd-n", "n", "No", true) }}
+                                {{ f.inlineRadio("rbrthd", "rbrthd-y", "y", "Yes", true) }}
+                            </div>
+                        </div>
+                        <div class="col-md-6">
+                            <label>Discharge location?</label>
+                            <div class="form-group">
+                                {{ f.inlineRadio("rbrdl", "rbrdl-h", "h", "Home", true) }}
+                                {{ f.inlineRadio("rbrdl", "rbrdl-s", "s", "Other Hospital", true) }}
+                                {{ f.inlineRadio("rbrdl", "rbrdl-t", "t", "Extended Care/Transitional Care Unit/Rehab", true) }}
+                                {{ f.inlineRadio("rbrdl", "rbrdl-n", "n", "Nursing Home", true) }}
+                                {{ f.inlineRadio("rbrdl", "rbrdl-p", "p", "Hospice", true) }}
+                                {{ f.inlineRadio("rbrdl", "rbrdl-o", "o", "Other", true) }}
```

Comment:
Typically, we have a textbox that goes along with these "other" options. So when user selects "other" we prompt (and require) a free-text entry. Do you know if that's necessary here?

Reply:
Their form-changes document showed no such field, and while I noticed the discrepancy, I took it as writ. Linda has apparently spoken with the upstream team and they've said they _do_ want a field there (but didn't give us a fieldname for it—I get to come up with it, which I'll notify all parties thereof), but they don't want it to be required. I'll add that.

Reply:
The new form-field name, by the way, is `rbrdls`, but I have a separate email to George and Artit pending on this.

Reply:
Fwiw, I named a lot of fields on my own. Elcap forms only got us so far, then I just named any new fields using a similar pattern.

Reply:
I should note that, per discussion with Linda, that field is not conditionally enabled; it may be used whatever checkbox is selected.

Reply:
I don't think that's a good idea. Selecting "other" should enable the new field. It doesn't have to be required if that's the direction you're given from ELCAP, but it should be conditionally enabled. It would make no sense for a user to select, say "Home" and also type into the other text box.